### PR TITLE
reset cart, delete cart item

### PR DESCRIPTION
**Pull request by dnpnghia**

### CODE REVIEW

**Comment by dnpnghia** [comment]
ready

**Comment by dnpnghia** [comment]
https://edu-redmine.sun-asterisk.vn/issues/41914

File: app/views/carts/index.html.erb [hunk 1]
```diff
@@ -29,9 +29,8 @@
                   locale: :vi) %></td>
                 <td>
                   <div class="d-flex flex-column">
-                    <a class="btn btn-outline-danger" href="#" role="button">
-                      <%= t "cart.delete" %>
-                    </a>
+                    <%= link_to t("cart.delete"), cart_path(book.id),
+                    method: "delete", class: "btn btn-outline-danger" %>
```
Comment (vanvtt-0952) [inline app/views/carts/index.html.erb]:
```suggestion
                      method: "delete", class: "btn btn-outline-danger" %>
```

**Comment by dnpnghia** [comment]
https://edu-redmine.sun-asterisk.vn/issues/41913

**Comment by dnpnghia** [comment]
ready

**Pull request merged**